### ISSUE TITLE
[Future Feature] Implement a "heated" table for culture jar

### ISSUE THREAD

**Issue by Alatyami** [GrowthcraftCE/Growthcraft-1.12#101]
Wood Burning Stove 

The wood burning stove, will have an input of burnable items that will result in a heated block that can then be used for the culture jar, brew kettle, or cheese vat. 

Function
You can add up to a full stack of logs or planks and then activate the stove with a flint and steel. The top of the stove will change color when it is producing heat.

By-Product
There will be a chance of producing charcoal (itemCoal) that can then be used in a furnace, but not the Wood Burning Stove.

**Comment by Avatair** [comment]
If a heated device is placed on the stove, then with the current design of the cheese vat and brewing kettle the player wouldn't be able to see the fluid inside anymore without jumping. Guess having this feature will require a smaller design for both devices.

**Comment by Alatyami** [comment]
Or a heated platform ... I wouldn't change the current devices.

**Comment by Alatyami** [comment]
While, I do agree, however, we already have a heavy GUI for the Brew Kettle, and the Culture Jar simply wouldn't make sense for having its own heat source.

**Comment by Alatyami** [comment]
The heat source block / device would open us up for actually adding on the cooking element that was planned for Minecraft 1.7.10. Keep in mind, fire and magma blocks would still be supported so this would be a tiered model. 

Fire > nether rack > magma block > Wood Burning Stove

**Comment by Nakuta** [comment]
I agree for the culture jar. Either a solid block that provides heat and looks like it makes sense to place a jar atop, or a rack that will hold the jar suspended over a heat source would be the best visual solutions.

**Comment by Syndaryl** [comment]
The culture jar would make sense if it included an iron rack/stand as part of the model; we have a old cast iron pot stand kicking around the place somewhere for cooking in a wood fireplace. I think it was for clay pots.
That might be an upgraded version of the culture jar, or a separate block that you put the culture jar in.
